src/wavm.h

```diff
@@ -34,7 +34,7 @@ class WavmEthereumInterface : public EthereumInterface {
 
 private:
   // These assume that m_wasmMemory was set prior to execution.
-  size_t memorySize() const override { return 0; } //TODO
+  size_t memorySize() const override { return m_wasmMemory->numPages * 65536; } //TODO 65536 bytes = 64 kb; not sure if this works
```

Comment:
This is used to check for overflow when reading/writing to the linear memory. As long as `memoryGet(memorySize())` is valid, this should be ok.